packages/rum-core/src/domain/rumEventsCollection/action/trackClickActions.ts

```diff
@@ -189,13 +232,24 @@ function newPotentialClickAction(
   }
 
   return {
+    base,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏it seems that we only need `base.event` outside of `potentialClickAction`, what about only exposing `event` then?